File: Python/TwoSum.py
def twoSum(nums, target):
    for i in range(0, len(nums), 1):
        sum = None
        if nums[i] < target:
            for j in range(0, len(nums), 1):
                if nums[j] < target:
                    sum = nums[i] + nums[j]
                    if sum == target:
                        break
        if sum == target:
            break
    return [i, j]


nums = [11, 25, 4, 5]
target = 9
x = twoSum(nums, target)
print(x)

File: C++/TwoSum.cpp
//#include <iostream>
//#include <vector>
//using namespace std;
//
//
//vector<int> twosum(vector<int> nums, int target) {
//	vector<int> out;
//	int check = 0;
//	for (int i = 0; i < nums.size(); i++) {
//		if (nums[i] < target){
//			for (int j = 0; j < nums.size(); j++) {
//				if (nums[j] < target) {
//					check = nums[i] + nums[j];
//					if (check == target) {
//						out.push_back(i);
//						out.push_back(j);
//						break;
//					}
//				}
//			}
//			if (check == target) {
//				break;
//			}
//		}
//	}
//	return out;
//}
//
//
//
//int main() {
//
//	vector<int> nums = { 11, 15, 2, 7 };
//	int target = 9;
//	vector<int> out;
//	out = twosum(nums, target);
//	for (auto i = out.begin(); i != out.end(); ++i)
//		cout << *i << " ";
//	return 0;
//}